pylxd/client.py

```diff
@@ -46,12 +46,12 @@ def __init__(self, api_endpoint, cert=None, verify=True, timeout=None):
 
     def __getattr__(self, name):
         return self.__class__(
-            '{}/{}'.format(self._api_endpoint, name),
+            '{}/{}'.format(self._api_endpoint, name.replace('_', '-')),
             cert=self.session.cert, verify=self.session.verify)
 
     def __getitem__(self, item):
         return self.__class__(
-            '{}/{}'.format(self._api_endpoint, item),
+            '{}/{}'.format(self._api_endpoint, item.replace('_', '-')),
```

Comment:
It's a shame to have to do this, but I can see why it's necessary now.  A comment would be useful for future reference.  (e.g. 'item' == the endpoint model 'e.g. storage-pools, profiles, images')